src/megaclient.cpp

```diff
@@ -10692,8 +10689,7 @@ bool MegaClient::startxfer(direction_t d, File* f, bool skipdupes)
             if (overquotauntil && overquotauntil > Waiter::ds)
             {
                 dstime timeleft = overquotauntil - Waiter::ds;
-                app->transfer_failed(t, API_EOVERQUOTA, timeleft);
-                t->bt.backoff(timeleft);
+                t->failed(API_EOVERQUOTA, timeleft);
```

Comment:
This block is also used for `Transfers` not found in the map of transfers, the `if` block of this `else`. We probably should change both.